tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "module": "commonjs",
+    "target": "es6",
+    "jsx": "preserve", 
+  },
+  "allowJs": false,
+  "noEmit": true,
+  "pretty": true,
+  "strictNullChecks": true,
```

Comment:
If we can we should just opt-in to `strict: true` which turns on all strict typing from the get go.